request-headers/app.py

```diff
@@ -0,0 +1,22 @@
+from flask import *
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return "Hello"
+
+
+@app.route("/content-type", methods=["POST"])
+def content_type():
+    # Content-Typeがapplication/jsonでないならerrorを返す
+    if not request.headers.get("Content-Type") == 'application/json':
+        # JSON形式でエラーを返す
+        return make_response(jsonify(res="error"), 400)
+    # リクエストから飛んできたContent-Typeを返す
+    return request.headers["Content-Type"]
```

Comment:
if u have a lot of time. I recommend practice using other headers. (e.g, `Authorization`)

and if u explain about `Authorization` header, plz talk about value format. (e.g, `Authorization: Bearer XXXX`) 
ref. https://openid-foundation-japan.github.io/rfc6750.ja.html (japanese)

Reply:
/cc @mt-coff